iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/webservice/v1/models/TwinPropertiesListApiModel.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+package com.microsoft.azure.iotsolutions.iothubmanager.webservice.v1.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.microsoft.azure.iotsolutions.iothubmanager.services.models.*;
+import com.microsoft.azure.iotsolutions.iothubmanager.webservice.v1.Version;
+
+import java.util.*;
+
+public final class TwinPropertiesListApiModel {
+
+    private final List<TwinPropertiesApiModel> items;
+    private String continuationToken;
+
+    public TwinPropertiesListApiModel(final TwinServiceListModel twins) {
+        this.items = new LinkedList<>();
+        this.continuationToken = twins.getContinuationToken();
+        for (TwinServiceModel t : twins.getItems()) {
+            this.items.add(new TwinPropertiesApiModel(t.getDeviceId(), t.getModuleId(), t.getProperties()));
+        }
+    }
+
+    @JsonProperty("Items")
+    public List<TwinPropertiesApiModel> getItems() {
+        return this.items;
+    }
+
+    @JsonProperty("$metadata")
+    public Dictionary<String, String> getMetadata() {
```

Comment:
Do you require Hashtable specifically?

Reply:
No, but all of our other code uses the same so I didn't want to change it here.